invoke/parser/parser.py

```diff
@@ -1,22 +1,43 @@
 import copy
+from typing import TYPE_CHECKING, Any, Iterable, List, Optional
 
 try:
     from ..vendor.lexicon import Lexicon
     from ..vendor.fluidity import StateMachine, state, transition
 except ImportError:
-    from lexicon import Lexicon
-    from fluidity import StateMachine, state, transition
+    from lexicon import Lexicon  # type: ignore
+    from fluidity import StateMachine, state, transition  # type: ignore
 
-from ..util import debug
 from ..exceptions import ParseError
+from ..util import debug  # type: ignore
 
+if TYPE_CHECKING:
+    from .context import ParserContext
 
-def is_flag(value):
-    return value.startswith("-")
 
+def is_flag(value: str) -> bool:
+    return bool(value.startswith("-"))
```

Comment:
Doesn't `str.startswith` always return True/False? Why the `bool`? (And ditto for the next, identical change.)

Reply:
This is cruft left over from troubleshooting if these strings were more complex types or plain strings. Cleaning it up now.